README.md

```diff
@@ -7,14 +7,20 @@ This repository contains playbooks and configuration to define a Slurm-based HPC
 - A monitoring backend using Prometheus and ElasticSearch.
 - Grafana with dashboards for both individual nodes and Slurm jobs.
 - Production-ready Slurm defaults for access and memory.
-- A Packer-based build pipeline for compute node images.
+- A Packer-based build pipeline for compute and login node images.
 
 The repository is designed to be forked for a specific use-case/HPC site but can contain multiple environments (e.g. development, staging and production). It has been designed to be modular and extensible, so if you add features for your HPC site please feel free to submit PRs back upstream to us!
 
-## Pre-requisites
+While it is tested on OpenStack it should work on any cloud, except for node rebuild/reimaging features which are currently OpenStack-specific.
 
-- Working DNS so that we can use the ansible inventory name as the address for connecting to services.
-- Bootable images based on Centos 8 Cloud images.
+## Prerequisites
+It is recommended to check the following before starting:
+- You have root access on the "ansible deploy host" which will be used to deploy the appliance.
```

Comment:
Why do we need root on the control host?

Reply:
- Packer build instructions require root for package installs on localhost.
- [Validation for app creds](https://github.com/stackhpc/ansible_collection_slurm_openstack_tools/blob/main/roles/rebuild/tasks/validate.yml#L28) requires sudo on localhost, doesn't work properly if it's not run with sudo, appears to be an `openstack.cloud.auth` module bug but 🤷.